src/liberty/devCommands.ts

```diff
@@ -610,4 +617,34 @@ async function getLocalGradleWrapper(projectFolder: string): Promise<string | un
  */
 function isWin(): boolean {
     return process.platform.startsWith("win");
-}
\ No newline at end of file
+}
+
+/*
+will return the path of the report, since there are diffrent folders to look into and the file names can be different 
+we need to get the paths to look for dynamically
+*/
+function getReportFile(path :any,dir :string,filename:string):any{
```

Comment:
Add space after comma and colon.
function getReportFile(path : any, dir :string, filename: string): any {

Reply:
Added spaces please see the changes in this [commit](https://github.com/JoshwinThomasIBM/liberty-tools-vscode/commit/327b975cf102358d31e8185180444adc9669afb2)